support/src/main/java/com/factual/cdh5docker/utils/ServiceLauncher.java

```diff
@@ -676,6 +670,7 @@ static void launchHBaseReadyStateThread(final LaunchConfig launchConfig, final C
       public void run() {
         Logger.getLogger("rg.apache.hadoop.hbase").setLevel(Level.ALL);
         try { 
+          waitForZookeeperToBeReady();
```

Comment:
Why do you need this since you already do the waitForZookeeperToBeReady before calling launchHBaseReadyStateThread ?